packages/components/tabs/src/Tab.tsx

```diff
@@ -1,35 +1,37 @@
 import React, { useCallback } from 'react';
-import classNames from 'classnames';
-import type { CSSProperties, KeyboardEvent, MouseEvent } from 'react';
+import type { KeyboardEvent, MouseEvent } from 'react';
+import type { CommonProps } from '@contentful/f36-core';
+import { Box } from '@contentful/f36-core';
 
-import styles from './Tabs.css';
+import { getTabStyles } from './Tabs.styles';
 
-export interface TabProps {
+export interface TabProps extends CommonProps {
   id: string;
   onSelect?: (id: string, e: React.SyntheticEvent) => void;
   selected?: boolean;
   href?: string;
   target?: string;
   disabled?: boolean;
   tabIndex?: number;
-  style?: CSSProperties;
-  className?: string;
-  testId?: string;
   children: React.ReactNode;
 }
 
-export function Tab({
-  children,
-  className,
-  disabled = false,
-  href,
-  id,
-  onSelect,
-  selected = false,
-  style,
-  tabIndex = 0,
-  testId = 'cf-ui-tab',
-}: TabProps): React.ReactElement {
+function _Tab(
+  {
+    children,
+    className,
+    disabled = false,
+    href,
+    id,
+    onSelect,
+    selected = false,
+    style,
+    tabIndex = 0,
+    testId = 'cf-ui-tab',
```

Comment:
it would be great to destuct `...otherProps` and apply them to the element as we do in all other places.